builtin/reset.c

```diff
@@ -100,9 +100,7 @@ static int reset_index(const struct object_id *oid, int reset_type, int quiet)
 
 	if (reset_type == MIXED || reset_type == HARD) {
 		tree = parse_tree_indirect(oid);
-		trace2_region_enter("exp", "prime_cache_tree", the_repository);
```

Comment:
Makes sense to revert this, but could you replace the commit subject

> Revert "trace2:gvfs:experiment: prime_cache_tree"

by

> fixup! trace2:gvfs:experiment: prime_cache_tree

so that the next rebase will automatically amend the original commit (so that the result becomes empty and will be dropped)?

Reply:
will do.

Reply:
done